Makefile

```diff
@@ -83,5 +83,40 @@ update-configmap:
 		--from-file=config.yaml=config-prod.yaml --dry-run=client -o yaml \
 			> k8s/configmap.yaml
 
+gauth:
+	@gcloud auth application-default print-access-token >/dev/null 2>&1 \
+		&& gcloud auth list --filter=status:ACTIVE --format="value(account)" | grep -q "@nav.no" \
+			&& echo "GCP ADC login not required." || gcloud auth login --update-adc
```

Comment:
Ikke så fan av at Make skal tukle med innloggingen min. Jeg vil heller ikke ha inn application-creds hvis jeg ikke trenger det. Kanskje enklere for meg å bare ikke kjøre de minikube-kommandoene her.

Reply:
Kan bare la være å bruke `run` kommandoen? Poenget er å kunne gjøre: `make clean && make run` og så ha et 100% legitimt miljø oppe og kjøre, og istedenfor at man dokumenterer via en README så har man kommandoene skrevet ned